go/billing/tests/test_admin.py

```diff
@@ -72,3 +75,20 @@ def test_account_admin_view(self):
         self.assertContains(response, 'Description')
         self.assertContains(response, 'Credit balance')
         self.assertContains(response, 'Last topup balance')
+        self.assertContains(response, 'Is developer')
+
```

Comment:
Yes, I wasn't sure exactly how to test this, so I tested the functions directly. Are there any examples of how to test Django admin actions?